pkg/util/watch_filters.go

```diff
@@ -43,6 +46,60 @@ func CAPIMachineToMAPIMachine(namespace string) func(context.Context, client.Obj
 	}
 }
 
+// RewriteNamespace takes a client.Object and returns a reconcile.Request for
+// it in the namespace provided.
+//
+// It is intended for use with CAPI Machines and MachineSet requests, where we expect
```

Comment:
I think CAPIMachineToMAPIMachine above should be replaced by RewriteNamespace, however I'm not doing that just yet as I want https://github.com/openshift/cluster-capi-operator/pull/212 to merge first, which relies on the other naming.

Reply:
212 is now merged so you should be able to fix this up now